dts/x86/arduino_101.fixup

```diff
@@ -1,7 +1,5 @@
-#define CONFIG_UART_QMSI_0_BAUDRATE	INTEL_ARDUNIO101_UART_B0002000_CURRENT_SPEED
```

Comment:
without sanity check i pushed for first time. It caused warning in build so removed and pushed it agin